packages/cli/src/databases/migrations/sqlite/1607431743769-MakeStoppedAtNullable.ts

```diff
@@ -2,15 +2,14 @@ import type { MigrationContext, IrreversibleMigration } from '@db/types';
 
 export class MakeStoppedAtNullable1607431743769 implements IrreversibleMigration {
 	async up({ queryRunner, tablePrefix }: MigrationContext) {
-		// SQLite does not allow us to simply "alter column"
```

Comment:
This migration was [first released in n8n@0.108.0](https://github.com/n8n-io/n8n/pull/1340/files#diff-5d0d33f218972ddb767224feefd11a0d1e03ef352bc83b2a7a2ac2f3f6d02a55).
If a instance is on an earlier version, and tried to upgrade to a version with this change, they'll lose all their executions.